tanner/tests/test_base.py

```diff
@@ -124,3 +124,53 @@ def test_set_injectable_page(self):
             sess = session.Session(None)
         injectable_page = self.handler.set_injectable_page(sess)
         self.assertEqual(injectable_page, '/python.html')
+
+    def test_emulate_type_1(self):
+        self.handler.set_injectable_page = mock.create_autospec(self.handler.set_injectable_page)
+        data = dict(method='GET', path='/index.html',
+                    cookies={'sess_uuid': '9f82e5d0e6b64047bba996222d45e72c'})
+
+        async def mock_handle_get(session, data):
+            return {'name': 'index', 'order': 2}
+
+        self.handler.handle_get = mock_handle_get
+        detection = self.loop.run_until_complete(self.handler.emulate(data, self.session))
+        assert_detection = {'name': 'index', 'order': 2, 'type': 1, 'version': '0.6.0'}
```

Comment:
Order for `index` should be 1, I guess. Also, don't hardcode the version otherwise tests will fail once we upgrade the version.

Reply:
Got it!